src/api.ts

```diff
@@ -14,24 +15,28 @@ export const getChapters = async () => {
   return camelizeKeys(payload) as { chapters: ChapterType[] };
 };
 
-export const getChapter = async (id: string | number | string[]) => {
+export const getChapter = async (id: string | number) => {
   const payload = await fetcher(makeUrl(`/chapters/${id}`));
 
   return camelizeKeys(payload) as { chapter: ChapterType };
 };
 
-export const getChapterInfo = async (id: string | number | string[]) => {
+export const getChapterInfo = async (id: string | number) => {
   const payload = await fetcher(makeUrl(`/chapters/${id}/info`));
 
   return camelizeKeys(payload);
 };
 
-export const getChapterVerses = async (id: string | number | string[]) => {
-  const payload = await fetcher(makeUrl(`/chapters/${id}/verses`, { translations: 20, limit: 25 })); // TODO (@abdellatif): parameterize the default translation
+export const getChapterVerses = async (id: string | number, params?: Record<string, unknown>) => {
```

Comment:
It's better if params is type safe. This way we know what params is supported by the backend